### PR TITLE
PE-419 - Adding support for contact id list.

### PR DESCRIPTION
**Pull request by Squirrel18**
## Description:

This PR adds support for converting the contact id, from a list object to a string object, taking only the first list value.

## Reviewers:

- [ ] @andrey-canon

### CODE REVIEW

File: lms/djangoapps/student_account/management/commands/migrate_contactid.py [hunk 1]
```diff
@@ -60,11 +60,18 @@ def handle(self, *args, **options):
             if not extra_data.get('contactid', False) or not getattr(social_user, 'provider', False):
```
Comment (andrey-canon) [inline lms/djangoapps/student_account/management/commands/migrate_contactid.py]: The `False` parameter is not necessary, is `contactid` a boolean ???

File: lms/djangoapps/student_account/management/commands/migrate_contactid.py [hunk 2]
```diff
@@ -60,11 +60,18 @@ def handle(self, *args, **options):
...
                 continue
 
+            user_contact_id = ''
```
Comment (andrey-canon) [inline lms/djangoapps/student_account/management/commands/migrate_contactid.py]: is this necessary ?

Comment (Squirrel18) [reply lms/djangoapps/student_account/management/commands/migrate_contactid.py]: Maybe don't.

**Comment by Squirrel18** [comment]
Addressed. @andrey-canon

File: lms/djangoapps/student_account/management/commands/migrate_contactid.py [hunk 3]
```diff
@@ -57,14 +57,21 @@ def handle(self, *args, **options):
 
             extra_data = getattr(social_user, 'extra_data', {})
 
-            if not extra_data.get('contactid', False) or not getattr(social_user, 'provider', False):
+            contact_id = extra_data.get('contactid', '')
```
Comment (Squirrel18) [inline lms/djangoapps/student_account/management/commands/migrate_contactid.py]: Contact id is a boolean value not a string one. @andrey-canon

**Review by andrey-canon — APPROVED** [review]

**Pull request merged**